pyproject.toml

```diff
@@ -4,6 +4,7 @@
 requires = [
     "Cython >=3.0.0,<4",
     "numpy >=2.0.0, <3",
+    "packaging >=23.0",
```

Comment:
is it actually used ?

Reply:
It is used by the commented out code.
I've removed the commented out code for now and I'll add a DNM PR once this is merged so that we don't forget where to re-add stuff.